.github/workflows/python-tests.yml

```diff
@@ -16,9 +16,11 @@ jobs:
       ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
I wouldn't do this

Reply:
For much slower CIs and much busier repos, that can be necessary, but it means other PRs (e.g. https://github.com/pypa/setuptools/pull/2414) don't get any results from GHA, and I think's usually it's more useful to get that info:

![image](https://user-images.githubusercontent.com/1324225/95755445-179d6a80-0cad-11eb-9449-b98de87829df.png)

For example, there might be a change that passes on all versions but is incompatible with Python 3.9. It's more useful to see 3.9 fail and the rest pass, to get better feedback. If 3.9 fails and the rest are cancelled, we don't know whether it's only a problem with 3.9 or problem for all.

But happy to remove it if not wanted.

Reply:
Yeah, I'm often presented with this dilemma too. I guess it's up to the maintainers. I've only wanted to point out that it takes ~10min to complete all jobs (https://github.com/pypa/setuptools/actions/runs/292445192) and you can still get most of that info from other CIs.

Reply:
My instinct is to use the default behavior unless there's a compelling reason to deviate.

Reply:
```suggestion
```